### PR TITLE
Locales: Update translations (fr-FR)

### PR DESCRIPTION
**Pull request by LaneSh4d0w**
Add some new translations, and fixing some.

### CODE REVIEW

**Comment by RedDevilus** [comment]
Where is the .mo file? Without it wont change the language.

**Comment by LaneSh4d0w** [comment]
Whoops. I'm going to look that up.

**Comment by RedDevilus** [comment]
https://my.pcloud.com/publink/show?code=kZDH81kZHH9txOzWRSfidhxYhMviSYe30lW7
How to help PCSX2 and update translation v6.pdf

You just need to compile to .mo

**Comment by LaneSh4d0w** [comment]
Yup, committing the changes to my fork.

**Comment by RedDevilus** [comment]
I would suggest reading that PDF. You are touching on English strings + po and mo have different locations.

**Comment by LaneSh4d0w** [comment]
The English Strings weren't modified, in fact i think PoEdit reformatted the way they were presented. I'll take the original file format and redo it again, it'll be easier.

**Comment by RedDevilus** [comment]
I believe you. The default settings of poedit are wrong.
image

Its all in the PDF  :)  Many have that problem so no worries.

**Comment by LaneSh4d0w** [comment]
Should be good. Changed POEdit preferences on mine.

**Comment by lightningterror** [comment]
Somehow it doesn't look right. Check line endings maybe?

**Comment by LaneSh4d0w** [comment]
I have this for the line endings. It is indeed weird.

image

**Comment by turtleli** [comment]
The correct Poedit settings depends on how you update the file and your git settings.

Try Unix line endings.

**Comment by LaneSh4d0w** [comment]
Made changes, by @turtleli 's request.

Files are now under UNIX line endings, 79 as usual.

**Comment by RedDevilus** [comment]
Looks good. Forgot that line endings settings from guide are only for windows users. My mistake.

File: locales/fr_FR/pcsx2_Iconized.po [hunk 1]
```diff
@@ -30,12 +30,11 @@ msgstr ""
 "été réservé par d'autres processus, services ou DLL."
 
 #: pcsx2/CDVD/CDVD.cpp:2058
-#, fuzzy
 msgid ""
 "Failed to read/write NVM/MEC file. Check your BIOS setup/permission settings."
 msgstr ""
-"Echec de l'écriture/lecture du fichier NMV/MEC Vérifiez vos paramètres/"
-"permissions dans votre bios."
+"Echec d'écriture/lecture du fichier NMV/MEC. Vérifiez vos paramètres/"
```
Comment (turtleli) [inline locales/fr_FR/pcsx2_Iconized.po:36]: NVM

**Comment by LaneSh4d0w** [comment]
@RedDevilus In fact i'm using a Windows computer. I wonder why that didn't work as expected.

**Comment by RedDevilus** [comment]
I'm using W10 x64. 🤔

**Comment by LaneSh4d0w** [comment]
I'm using the same thing. Might be something else then. I do not know.

File: locales/fr_FR/pcsx2_Iconized.po (hunk 1 above)
Comment (turtleli) [reply locales/fr_FR/pcsx2_Iconized.po:36]: I suppose I wasn't clear. It's NVM, not NMV.

**Comment by turtleli** [comment]
It's to do with how you update the file, and your git settings (the autocrlf setting) if that's relevant to the update method.

File: locales/fr_FR/pcsx2_Iconized.po (hunk 1 above)
Comment (LaneSh4d0w) [reply locales/fr_FR/pcsx2_Iconized.po:36]: Oh okay. Lemme change it. I thought you said nevermind.

**Comment by LaneSh4d0w** [comment]
Got it.

**Comment by lightningterror** [comment]
Need to update the mo files again.

**Comment by LaneSh4d0w** [comment]
All set.

**Comment by atomic83GitHub** [comment]
"rapport de forme" should be "Ratio de l'image".

File: locales/fr_FR/pcsx2_Iconized.po [hunk 2]
```diff
@@ -622,32 +630,44 @@ msgid ""
...
 "demanding games, slows down others."
 msgstr ""
+"75% cyclerate. Petite réduction des exigences liées au CPU. Amélioration des "
+"performances sur des jeux moins demandeurs, ralentit les autres."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:47
 msgid ""
 "Default cyclerate. Runs the emulated PS2 Emotion Engine at normal speed."
 msgstr ""
+"Taux de cycle par défaut. Fait tourner le PS2 Emotion Engine à vitesse "
```
Comment (atomic83GitHub) [inline locales/fr_FR/pcsx2_Iconized.po:660]: Nombre de cycles par défaut.

File: locales/fr_FR/pcsx2_Iconized.po [hunk 3]
```diff
@@ -656,6 +676,9 @@ msgid ""
 "games will have higher internal framerates. FMVs may be slow. May cause "
 "stability problems."
 msgstr ""
+"180% cyclerate. Augmentation significative des exigences liées au CPU. Les "
+"jeux ayant un nombre d'images variable bénéficieront d'une augmentation. Les "
+"FMV peuvent être lents. Peut causer des problèmes de stabilités."
```
Comment (atomic83GitHub) [inline locales/fr_FR/pcsx2_Iconized.po:681]: Les cinématiques peuvent être lentes.

File: locales/fr_FR/pcsx2_Iconized.po [hunk 4]
```diff
@@ -664,12 +687,14 @@ msgid ""
 "games will have higher internal framerates. FMVs may be slow. May cause "
 "stability problems."
 msgstr ""
+"300% cyclerate. Augmentation extrême des exigences liées au CPU. Les jeux "
+"ayant un nombre d'images par secondes variable bénéficieront d'une "
+"augmentation. Les FMV peuvent être lents. Peut causer des problèmes de "
```
Comment (atomic83GitHub) [inline locales/fr_FR/pcsx2_Iconized.po:692]: Les cinématiques peuvent être lentes.

**Comment by LaneSh4d0w** [comment]
Could be debatable. Both are acceptable. "Ratio de l'image" is more technical, while "Rapport de forme" is an abstracted way of saying the same. I have changed it in case of, but just to let you know that it is possible to say both.

File: locales/fr_FR/pcsx2_Iconized.po [hunk 5]
```diff
@@ -622,32 +630,45 @@ msgid ""
 "50% cyclerate. Significant reduction of CPU requirements. Speedup for very "
 "lightweight games, slows down others. FMVs and audio may stutter or skip."
 msgstr ""
+"50% cyclerate. Réduction significative des exigences liées au CPU. "
+"Amélioration des performances sur des très petits jeux, ralentit les autres. "
```
Comment (atomic83GitHub) [inline locales/fr_FR/pcsx2_Iconized.po:634]: Should be: "Amélioration des performances sur des jeux non gourmand".

**Comment by atomic83GitHub** [comment]
I am okay with the new changes thanks, just one remaining and we should be good.

**Comment by LaneSh4d0w** [comment]
Should be okay now. Feel free to look at the new translations I added. MO file updated.

**Comment by lightningterror** [comment]
Presets tooltip isn't translated, shall I merge it like this anyway?

**Comment by LaneSh4d0w** [comment]
I suppose merging it as-is is good.

**Pull request merged**